corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -2,80 +2,101 @@ hqDefine("geospatial/js/geospatial_map", [
     "jquery",
     "hqwebapp/js/initial_page_data",
     "knockout",
-    "hqwebapp/js/bootstrap3/alert_user",
 ], function (
     $,
     initialPageData,
-    ko,
-    alert_user
+    ko
 ) {
     const defaultMarkerColor = "#808080"; // Gray
+    const defaultUserMarkerColor = "#0e00ff"; // Blue
```

Comment:
Nothing blocking, just curious how these colours were chosen? Is there a specification somewhere?

Reply:
@Charl1996 The marker colours aren't mentioned in the spec, so I chose a random colour to differentiate between cases/users.

@mkangia The `defaultMarker` is currently only used by cases, so I think this can be renamed to `defaultCaseMarker` to better state what it is being used for.

Reply:
Sounds good.